src/components/CIPipelineN/CustomInputVariableSelect.tsx

```diff
@@ -93,14 +93,21 @@ function CustomInputVariableSelect({ selectedVariableIndex }: { selectedVariable
                 },
             ])
         } else {
+            const excludeVariables = ["DOCKER_IMAGE_TAG", "DOCKER_IMAGE"]
+            let _globalVariables = []
+            globalVariables.forEach((val) => {
+                if(!excludeVariables.includes(val.value)){
+                    _globalVariables.push(val)
+                }
+            })
             setInputVariableOptions([
                 {
                     label: 'From Previous Steps',
                     options: previousStepVariables,
                 },
                 {
                     label: 'System variables',
-                    options: isCdPipeline ? globalVariables.filter((variable) => variable.stageType !== 'post-cd') : globalVariables ,
+                    options: isCdPipeline ? _globalVariables.filter((variable) => variable.stageType !== 'post-cd') : _globalVariables ,
```

Comment:
we can exclude check in this filter only, no need to create ne variable

Reply:
done